### PR TITLE
React Capstone Project

### PR DESCRIPTION
**Pull request by Bennyjoez**
# React Capstone Project 🗿 📦 
I have managed to implement a number of features in this project: 
 ### Setup ⚙️ ⚙️ 
- I created the app using CRA.✔️
- Linked my project to GitHub. ✔️
- Followed git-flow ✔️
- Installed linters for React and Redux.✔️
- Created the formal document structure.✔️
- Set up redux and the redux store in my App. ✔️
- Consumed an API to get the data and displayed it as an MVP.✔️
- Set up react-router.✔️
- Data filtering. ✔️ 
### Styling 🎨 :art
- I used regular CSS to style the components. ✔️ 
- I fixed the theme color provided from the designs. 🎨 
- I added a background image to the cards and header sections. 🪦 
### Testing 🧪 
I also tested the following components and got all green 🟢 
- App component. 🧪 
- Navigation component 🧪 
- Home component  🧪 
- Details component  🧪 
- Homecard component  🧪 
### Documentation 📝 
I also update the Readme with the essential data like:
- Live video walkthrough
- Live link
- License
- Features
- Authors
- Future features.

### CODE REVIEW

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Bennyjoez   , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] No linter errors
- [x] Nice UI design💯 
- [x] Tests are passing 💯 

 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

File: README.md [hunk 1]
```diff
@@ -1,70 +1,180 @@
...
+If you like this project, you can show your support by giving this project's repository a star on [github](https://github.com/microvese-projects/crypto-metrics/). 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the outstanding collection of information they have provided that enabled me to create the project.
+
```
Comment (KayLemba) [inline README.md:175]: It says in the project requirements that The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by KayLemba — CHANGES_REQUESTED** [review]

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Bennyjoez   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉

- [x] No linter errors. ✔️

- [x] Gitflow is followed. ✔️
- [x] Nice UI design 
- [x] Tests are passing 

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (KayLemba) [inline README.md]: It says in the project requirements that The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Bennyjoez   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉

- [x] No linter errors. ✔️

- [x] Gitflow is followed. ✔️
- [x] Nice UI design 
- [x] Tests are passing 

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Bennyjoez** [comment]
@KayLemba 
Hello Good Sir, I have seen several contradicting reviews. What changes would you like me to effect kindly? I am lost 😃 I can also engage in a call if that would help.

**Comment by KayLemba** [comment]
> @KayLemba Hello Good Sir, I have seen several contradicting reviews. What changes would you like me to effect kindly? I am lost 😃 I can also engage in a call if that would help.

They are not contradicting reviews its an error I made I think you can read in the comments section what you missed out on if you are unsure call me on zoom.

**Comment by KayLemba** [comment]
You can call me on zoom if you are unsure

File: README.md (hunk 1 above)
Comment (Bennyjoez) [reply README.md:175]: @KayLemba Changes have been effected and pushed. Thank you. 🙏🏾

**Comment by Bennyjoez** [comment]
Changes effected and pushed @KayLemba

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Bennyjoez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did a great job of acknowledging the author for the designs used as requested by the previous reviewer.

## Optional suggestions

- [ ] Kindly note that a good GitHub workflow should always be followed as per project requirements. In this case, Gitflow was supposed to be used and not GitHub flow. You should have created a feature branch from a development branch.

gitflow


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KayLemba** [comment]
> Hi 👋 @Bennyjoez,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉
> ## Highlights
> 
> 👍 You did a great job of acknowledging the author for the designs used as requested by the previous reviewer.
> ## Optional suggestions
> 
>     * [ ]  Kindly note that a good GitHub workflow should always be followed as per project requirements. In this case, Gitflow was supposed to be used and not GitHub flow. You should have created a feature branch from a development branch.
> 
> 
> gitflow
> 
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers, and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@elisha2kyakpo1 the student used correct gitflow he created seperate branches and merged all of them in the development branch so he compared the main branch and development branch which is his default branch for the final submission

**Comment by Bennyjoez** [comment]
Screenshot from 2023-05-31 14-03-07

Hello @elisha2kyakpo1 
I used git-flow for this project. This can be confirmed on the screenshot sent. 🏁 The default branch is the develop branch. and I have been making my changes to feature branches and then merging them to develop. @elisha2kyakpo1 

Thank you for approving my project. 👍🏾

**Comment by Bennyjoez** [comment]
Thank you Sir @KayLemba

**Pull request merged**